tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/management/BrokerBalancerControlTest.java

```diff
@@ -141,6 +141,27 @@ public void testGetLocalTarget() throws Exception {
 
       CompositeData connectorData = (CompositeData)targetData.get("connector");
       Assert.assertNull(connectorData);
+
+      assertNull(brokerBalancerControl.getLocalTargetFilter());
```

Comment:
why not creating other tests for those new BrokerBalancerControl methods?

Reply:
would be better, thanks!